metadata.yaml

```diff
@@ -1,50 +1,31 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file populates the Overview on Charmhub.
-# See https://juju.is/docs/sdk/metadata-reference for a checklist and guidance.
-
-# The charm package name, no spaces (required)
-# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
-name: is-charms-template
- 
-# The following metadata are human-readable and will be published prominently on Charmhub.
-
-# (Recommended)
-display-name: Charm Template
-
-# (Required)
-summary: A very short one-line summary of the charm.
+name: smtp-integrator
+display-name: SMTP Integrator
+summary: SMTP Integrator to centralize SMTP configuration.
 docs: https://discourse.charmhub.io
-issues: https://github.com/canonical/is-charms-template-repo/issues
+issues: https://github.com/canonical/smtp-integrator-operator/issues
 maintainers: 
   - https://launchpad.net/~canonical-is-devops
-source: https://github.com/canonical/is-charms-template-repo
-
+source: https://github.com/canonical/smtp-integrator-operator/issues
 description: |
-  A single sentence that says what the charm is, concisely and memorably.
-
-  A paragraph of one to three short sentences, that describe what the charm does.
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+  deploying and managing a SMTP Integrator on Kubernetes and bare metal. SMTP is a communication
```

Comment:
More a question, is it relevant to run this charm as a machine charm ?

Reply:
It might be for the operator if deploying everything on VMs/bare metal

Reply:
Then we should probably rephrase the next sentences which are very focused on K8S